openfisca_uk/variables/finance/benefit/family/tax_credits.py

```diff
@@ -50,16 +50,14 @@ def formula(benunit, period, parameters):
 class is_CTC_child_limit_exempt(Variable):
     value_type = bool
     entity = Person
-    label = u"Exemption from Child Tax Credit two-child limit"
-    documentation = (
-        "Exemption from Child Tax Credit two-child limit based on birth year"
-    )
+    label = u"Exemption from Child Tax Credit child limit"
+    documentation = "Exemption from Child Tax Credit limit on number of children based on birth year"
     definition_period = YEAR
 
     def formula(person, period, parameters):
         limit_year = parameters(
             period
-        ).benefit.tax_credits.child_tax_credit.two_child_limit_year
+        ).benefit.tax_credits.child_tax_credit.limit.start_year
         return person("birth_year", period.this_year) <= limit_year
```

Comment:
I didn't get a chance to add legislative references here, but this differs from the equivalent UC code:
```python
born_before_limit = person("birth_year", period) < start_year
```
`period` vs `period.this_year` and `<` vs `<=`

Reply:
@nikhilwoodruff this is the only lingering question in this PR

Reply:
OK - since it's yearly, `period === period.this_year` (so fine either way IMO, though `period` would be shorter). On the `<` issue - we are already simplifying from the law: in reality, the birth date needs to be before April 2017. So if it's just a question of this operation is closer to the law, `<` is better.

Reply:
OK updated to match UC. This made us further from the administrative total though, so I had to increase the tolerance from 7bn to 8bn to pass tests.